common/pipeline_storage/src/main/scala/uk/ac/wellcome/pipeline_storage/ElasticIndexer.scala

```diff
@@ -72,13 +73,26 @@ class ElasticIndexer[T: Indexable](
 
               // Slice the documents in two, and index them both separately.
               // We'll combine the results.
+              //
+              // The "trace ID" allows us to follow multiple requests through the logs.
```

Comment:
Good idea

Reply:
I'm honestly tempted to rip all of this logic out and just have it bubble up as an error, but this'll do for now.